cluster/addons/fluentd-elasticsearch/fluentd-es-ds.yaml

```diff
@@ -1,20 +1,20 @@
 apiVersion: extensions/v1beta1
 kind: DaemonSet
 metadata:
-  name: fluentd-es-v1.22
+  name: fluentd-es
   namespace: kube-system
   labels:
     k8s-app: fluentd-es
     kubernetes.io/cluster-service: "true"
     addonmanager.kubernetes.io/mode: Reconcile
-    version: v1.22
+    version: v1.23
```

Comment:
I think the current version is 24?  probably shouldn't change it yet as there may be other PRs that bump the version but something to check when closer to merge.